onnx_coreml/converter.py

```diff
@@ -411,7 +411,7 @@ def convert(model,  # type: Union[onnx.ModelProto, Text]
         BNBroadcastedAddFuser(),
         ReshapeTransposeReshape_pattern1(),
         PixelShuffleFuser(),
-        AddModelInputsOutputs(),
+        # AddModelInputsOutputs(),
```

Comment:
can we conditionally disbale it if ``disable_rank_5`` is true

Reply:
thought of embedding `disable_rank_5` inside graph so that can skip optimization within with early exit. But that approach does not seems to be scalable as we will need to keep adding such options in future or refactor again when we completely disable rank5 in future.

Instead introducing `dummy_transformation` which does nothing and will call to it instead of intended disabled transformation